src/core/positioning/tcpreceiver.cpp

```diff
@@ -119,7 +119,7 @@ void TcpReceiver::handleError( QAbstractSocket::SocketError error )
       mLastError = tr( "The connection was refused by the remote host" );
       break;
     default:
-      mLastError = tr( "UDP receiver error (%1)" ).arg( QMetaEnum::fromType<QAbstractSocket::SocketError>().valueToKey( error ) );
+      mLastError = tr( "TCP receiver error (%1)" ).arg( QMetaEnum::fromType<QAbstractSocket::SocketError>().valueToKey( error ) );
```

Comment:
In this function also mSocketState and socket state string should be updated

Reply:
Let's defer to different PR here to focus on egeniouss bits